src/form-builder/TextField.tsx

```diff
@@ -35,17 +35,7 @@ const Wrapper = (props: FieldProps<string>) => {
 
 const TextField = (props: FieldProps<string>): JSX.Element => {
   const id = props.id || props.name;
-  const validate = (value: string) => {
-    if (props.required && !value) {
-      const errorMessage =
-        typeof props.required === 'string'
-          ? props.required
-          : getMessage('required.default');
-      return errorMessage;
-    }
-
-    return props.validate ? props.validate(value) : undefined;
-  };
+  const validate = chainValidations(props, [required]);
```

Comment:
Are you referring to the `validate` function? In `master`, that function only adds the `required` validation. I merged in `master` and changed it up a bit so that `required` validation is now called `required` and is chainable using this `chainValidations`.

Reply:
I think I left this comment on a bad spot - I was referring to the other non-chaining changes (try saying that three times fast) that I had...changed... in another PR.

Reply:
:joy: :+1: